scripts/functions/requirements/freebsd

```diff
@@ -105,6 +111,17 @@ requirements_freebsd_define()
     (opal)
       requirements_check_custom node
       ;;
+    (rbx*)
+      if
+        __rvm_version_compare ${_system_version} -ge 10.0
+      then
+        requirements_check gmake llvm35
+        rvm_warn "If rbx installation fails, build it with clang and make sure llvm-config is in your PATH"
+        rvm_warn "Pass \"--cc clang --cxx clang++\" to rvm install to build with clang on FreeBSD"
+      else
+        rvm_warn "rbx installation by rvm is only supported on FreeBSD 10 or higher"
+      fi
+      ;;
```

Comment:
https://github.com/rys/rvm/blob/freebsd-10/scripts/functions/requirements/ubuntu#L99-L108 - maybe just add the packages and set variables?